Database/create.sql

```diff
@@ -0,0 +1,95 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema rollcall
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema rollcall
+-- -----------------------------------------------------
+DROP SCHEMA IF EXISTS `rollcall`;
+CREATE SCHEMA IF NOT EXISTS `rollcall` DEFAULT CHARACTER SET utf8 ;
+USE `rollcall` ;
+
+-- -----------------------------------------------------
+-- Table `rollcall`.`users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `rollcall`.`users` (
+  `user_id` INT NOT NULL AUTO_INCREMENT,
+  `user_role` ENUM("TEACHER", "STUDENT") NOT NULL,
+  `email` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(100) NOT NULL,
+  `first_name` VARCHAR(45) NOT NULL,
+  `last_name` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`user_id`),
+  UNIQUE INDEX `user_id_UNIQUE` (`user_id` ASC) VISIBLE,
+  UNIQUE INDEX `email_UNIQUE` (`email` ASC) VISIBLE)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `rollcall`.`classes`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `rollcall`.`classes` (
+  `class_id` INT NOT NULL AUTO_INCREMENT,
+  `name` VARCHAR(45) NOT NULL,
+  PRIMARY KEY (`class_id`),
+  UNIQUE INDEX `name_UNIQUE` (`name` ASC) VISIBLE)
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `rollcall`.`teachers_classes`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `rollcall`.`teachers_classes` (
```

Comment:
just an idea: To rename this table to "lectures" to make it more readable?

Reply:
Might be necessary as we might need classes for something else. Might need to add table for class like SD22w and assign students and lectures to it, as it says that teacher needs statistics either about individual students or whole classes. So we either add that or query attendance based on start/end date but it won't work that much for scalability in case schools have multiple classes as we had also here with 3 danish and 1 international before.